extensions/wikia/WikiaMiniUpload/WikiaMiniUpload_body.php

```diff
@@ -540,95 +554,95 @@ function insertImage() {
+					$file = new LocalFile( $title, RepoGroup::singleton()->getLocalRepo() );
 
 					if ( !empty($extraId) ) {
 						$flickrResult = $this->getFlickrPhotoInfo( $extraId );
 
-						$nsid = $flickrResult['owner']['nsid']; // e.g. 49127042@N00
-						$username = $flickrResult['owner']['username']; // e.g. bossa67
-						$license = $flickrResult['license'];
+						$nsid = $flickrResult[ 'owner' ][ 'nsid' ]; // e.g. 49127042@N00
+						$username = $flickrResult[ 'owner' ][ 'username' ]; // e.g. bossa67
+						$license = $flickrResult[ 'license' ];
 
-						$caption = '{{MediaWiki:Flickr'.intval($license).'|1='.wfEscapeWikiText($extraId).'|2='.wfEscapeWikiText($nsid).'|3='.wfEscapeWikiText($username).'}}';
+						$caption = '{{MediaWiki:Flickr' . intval( $license ) . '|1=' . wfEscapeWikiText( $extraId ) . '|2=' . wfEscapeWikiText( $nsid ) . '|3=' . wfEscapeWikiText( $username ) . '}}';
 					} else {
 						// get the supplied license value
 						$license = $wgRequest->getVal( 'ImageUploadLicense' );
 
 						if ( $license != '' ) {
-							$caption = '== ' . wfMsgForContent( 'license' ) . " ==\n" . '{{' . $license . '}}' . "\n";
+							$caption = '== ' . wfMessage( 'license' )->inContentLanguage()->text() . " ==\n" . {{' . $license . '}} . "\n";
 						} else {
-							$caption = "";
+							$caption = '';
 						}
 					}
 
-					$file->upload($temp_file->getPath(), '', $caption);
+					$file->upload( $temp_file->getPath(), '', $caption );
 					$temp_file->delete('');
 					$this->tempFileClearInfo( $tempid );
 				}
 
 				if ( $wgUser->getGLobalPreference( 'watchdefault' ) || ( $newFile && $wgUser->getGlobalPreference( 'watchcreations' ) ) ) {
-					$wgUser->addWatch($title);
+					$wgUser->addWatch( $title );
 				}
 				$db =& wfGetDB(DB_MASTER);
 				$db->commit();
 			}
 		} else {
-			$title = Title::newFromText($mwname, 6);
+			$title = Title::newFromText( $mwname, NS_FILE );
 		}
 
-		if ( is_null($file) ) {
+		if ( is_null( $file ) ) {
 			$file = wfFindFile( $title );
 		}
 
-		if ( !is_object($file) ) {
-			header('X-screen-type: error');
-			return wfMessage('wmu-file-not-found')->plain();
+		if ( !is_object( $file ) ) {
+			header( 'X-screen-type: error' );
+			return wfMessage( 'wmu-file-not-found' )->plain();
 		}
 
 		// Test if this violates the size requirements we've been given
-		if ( $msg = $this->invalidSize($file) ) {
-			header('X-screen-type: error');
+		if ( $msg = $this->invalidSize( $file ) ) {
+			header( 'X-screen-type: error' );
 			return $msg;
 		}
 
-		$ns_img = $wgContLang->getFormattedNsText( NS_IMAGE );
+		$ns_img = $wgContLang->getFormattedNsText( NS_FILE );
 
 		if ( ( -2 == $gallery ) && !$ck ) {
 			// this went in from the single placeholder...
-			$name = $title->getText();
-			$size = $wgRequest->getVal('size');
-			$width = $wgRequest->getVal('width');
-			$layout = $wgRequest->getVal('layout');
+			$name = $title->getplain();
```

Comment:
@KockaAdmiralac  - there's no `getplain` method on `Title` class object.

Reply:
Oh, that might've been an error in find and replace :thinking: 
Should I submit a pull request to the new branch to fix that?

Reply:
Yes, please